packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/RealmInstant.kt

```diff
@@ -73,6 +74,11 @@ public interface RealmInstant : Comparable<RealmInstant> {
             }
             return RealmInstantImpl(s, ns)
         }
+
+        public fun now(): RealmInstant {
```

Comment:
This method seems to be missing documentation. Generally we strive to document all of our public API methods=